src/app/services/products.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+import {Product} from "../model/product";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProductsService {
+
+  productsURL = 'http://localhost:3000/products';
```

Comment:
**suggestion:** you cand keep this url inside the `environment.ts` file or you could use a [proxy](https://angular.io/guide/build#proxying-to-a-backend-server)